package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "typeorm",
   "private": true,
-  "version": "0.0.9",
+  "version": "0.1.0-alpha.1",
```

Comment:
Acutally, it's a semver 😄 There is a special treatment for 0.x.x releases, as you said - 0.MAJOR.MINOR. npm also supports it, however for releases 0.0.x it always stick to the version (like 0.0.12), but when `^0.1.0` it will accept all 0.1.x releases:
https://nodesource.com/blog/semver-tilde-and-caret/
So all is ok, for rapid development you can continue using 0.0.x (so no 0.1.0 just 0.0.10) if TypeORM is in early stage and you plan to do a lot of API changes in near feature 😉 npm will keep installing 0.0.9 for `^0.0.9` even if there is 0.0.23 or 0.5.12 available